examples/3d-tiles/app.js

```diff
@@ -224,23 +232,8 @@ export default class App extends PureComponent {
 
   // Called by Tile3DLayer when a new tileset is load
   _onTilesetLoad(tileset) {
-    if (!this._tilesetStatsWidget) {
-      // TODO - would be nice to be able to create stats widget in constructor without stats object
-      // TODO - need method to detach stats widget in unmount...
-      this._tilesetStatsWidget = new StatsWidget(tileset.stats, {
-        container: this._statsWidgetContainer,
-        framesPerUpdate: 1,
-        // TODO - This is noisy. Default formatters should already be pre-registered on the stats object
-        formatters: {
-          Points: 'memory',
-          'Tile Memory Use': 'memory'
-        }
-      });
-    } else {
-      // TODO - this hack "works" but does not update the stats widget title
-      this._tilesetStatsWidget.stats = tileset.stats;
-      this._updateStatWidgets();
-    }
+    this._tilesetStatsWidget.setStats(tileset.stats);
```

Comment:
Shouldn't `setStats` trigger an `update` automatically?

Again I would like the addition of a StatsWidget to intrude minimally on the code.

Reply:
https://github.com/uber-web/probe.gl/pull/93/files